akka-remote/src/main/scala/akka/remote/artery/ArteryTransport.scala

```diff
@@ -369,6 +381,18 @@ private[remote] class ArteryTransport(_system: ExtendedActorSystem, _provider: R
     attachStreamRestart("Inbound message stream", completed, () ⇒ runInboundOrdinaryMessagesStream())
   }
 
+  private def runInboundLargeMessagesStream(): Unit = {
+    if (largeMessageDestinationsEnabled) {
+      // TODO just cargo-cult programming here
+      val completed = Source.fromGraph(new AeronSource(inboundChannel, largeStreamId, aeron, taskRunner, envelopePool))
```

Comment:
Ah, yes, `ArteryTransport.MaximumFrameSize` which is currently 1Mb, so the large one fits in there. Will add a separate config.